<launch>

    <node name="rec_console" pkg="sitl_dvrk" type="rec_console.py" output="screen">
        <param name="save_dir" type="str"    value="/home/$(env USER)/glove_console_rec"/>
    </node>

    <?ignore
    
    ?>

</launch>